containers/registry-mock-server/openapi/src/index.ts

```diff
@@ -192,8 +192,9 @@ app.all('/mock/*', (req: Request, res: Response) => {
     );
   }
   res.locals.projectId = parts[1];
-  res.locals.locationId = parts[5];
-  res.locals.apiId = parts[7];
+  res.locals.locationId = parts[3];
```

Comment:
@ssvaidyanathan We are expecting the URLs to be in the format
https://mock-server.client.com/mock/projects/projectid/locations/global/apis/apiId/versions/versionId/spec/SpecId

What was the URL pattern you were using?